include/osw_hal.h

```diff
@@ -82,6 +82,7 @@ class OswHal {
     void requestEnableDisplayBuffer();
     void disableDisplayBuffer();
     void enableDisplayBuffer();
+    bool displayBufferEnabled();
```

Comment:
you mean ... `OswHal::getInstance()->enableDisplayBuffer();`(https://github.com/Open-Smartwatch/open-smartwatch-os/blob/bc08af49ac33bde2c81f463ee0a1be0812b8d388/src/apps/tools/ble_media_ctrl.cpp#L64)?